data/configs/dnsmasq.conf

```diff
@@ -12,6 +12,7 @@ cache-size=0
 
 # These zones have their own DNS server
 server=/ipa.test/172.16.100.10
+server=/ipa2.test/172.16.100.80
```

Comment:
It does not matter much but you can use `.11` instead of `.80` to keep IPA servers grouped together.

Reply:
Fixed, switched to `172.16.100.11`